src/ui/utils/utils.go

```diff
@@ -132,19 +123,13 @@ func TriggerImageScan(repository string, tag string) error {
 }
 
 // NewRepositoryClientForUI ...
-// TODO need a registry client which accept a raw token as param
-func NewRepositoryClientForUI(endpoint string, insecure bool, username, repository string,
-	scopeActions ...string) (*registry.Repository, error) {
-
-	authorizer := auth.NewRegistryUsernameTokenAuthorizer(username, "repository", repository, scopeActions...)
-	store, err := auth.NewAuthorizerStore(endpoint, insecure, authorizer)
+func NewRepositoryClientForUI(username, repository string) (*registry.Repository, error) {
```

Comment:
Please explicitly point out in comment that this client is for access internal registry.